<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WordEditor</class>
 <widget class="QMainWindow" name="WordEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WordEditor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="file_menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="new_action"/>
    <addaction name="open_action"/>
    <addaction name="separator"/>
    <addaction name="save_action"/>
    <addaction name="saveAs_action"/>
    <addaction name="separator"/>
    <addaction name="options_action"/>
    <addaction name="exit_action"/>
   </widget>
   <widget class="QMenu" name="edit_menu">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="undo_action"/>
    <addaction name="redo_action"/>
    <addaction name="separator"/>
    <addaction name="cut_action"/>
    <addaction name="copy_action"/>
    <addaction name="paste_action"/>
    <addaction name="delete_action"/>
    <addaction name="separator"/>
    <addaction name="findAndReplace_action"/>
   </widget>
   <widget class="QMenu" name="insert_menu">
    <property name="title">
     <string>插入</string>
    </property>
    <addaction name="picture_action"/>
   </widget>
   <addaction name="file_menu"/>
   <addaction name="edit_menu"/>
   <addaction name="insert_menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="save_action"/>
   <addaction name="separator"/>
   <addaction name="undo_action"/>
   <addaction name="redo_action"/>
   <addaction name="separator"/>
   <addaction name="cut_action"/>
   <addaction name="paste_action"/>
   <addaction name="separator"/>
   <addaction name="left_align_action"/>
   <addaction name="center_align_action"/>
   <addaction name="right_align_action"/>
   <addaction name="justify_align_action"/>
   <addaction name="separator"/>
   <addaction name="bold_action"/>
   <addaction name="italics_action"/>
   <addaction name="underline_action"/>
  </widget>
  <action name="new_action">
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="open_action">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="save_action">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/save.png</normaloff>:/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="saveAs_action">
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="options_action">
   <property name="text">
    <string>选项</string>
   </property>
  </action>
  <action name="exit_action">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="undo_action">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/undo.png</normaloff>:/icons/undo.png</iconset>
   </property>
   <property name="text">
    <string>撤销</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="redo_action">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/redo.png</normaloff>:/icons/redo.png</iconset>
   </property>
   <property name="text">
    <string>重做</string>
   </property>
  </action>
  <action name="cut_action">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/cut.png</normaloff>:/icons/cut.png</iconset>
   </property>
   <property name="text">
    <string>剪切</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="copy_action">
   <property name="text">
    <string>复制</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="paste_action">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/paste.png</normaloff>:/icons/paste.png</iconset>
   </property>
   <property name="text">
    <string>粘贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="delete_action">
   <property name="text">
    <string>删除</string>
   </property>
  </action>
  <action name="findAndReplace_action">
   <property name="text">
    <string>查找/替换</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="picture_action">
   <property name="text">
    <string>图片</string>
   </property>
  </action>
  <action name="left_align_action">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/left.png</normaloff>:/icons/left.png</iconset>
   </property>
   <property name="text">
    <string>左对齐</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="center_align_action">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/middle.png</normaloff>:/icons/middle.png</iconset>
   </property>
   <property name="text">
    <string>居中对齐</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="right_align_action">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/right.png</normaloff>:/icons/right.png</iconset>
   </property>
   <property name="text">
    <string>右对齐</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="justify_align_action">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/just.png</normaloff>:/icons/just.png</iconset>
   </property>
   <property name="text">
    <string>两端对齐</string>
   </property>
   <property name="toolTip">
    <string>两端对齐</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="bold_action">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/bold.png</normaloff>:/icons/bold.png</iconset>
   </property>
   <property name="text">
    <string>粗体</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="italics_action">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/i.png</normaloff>:/icons/i.png</iconset>
   </property>
   <property name="text">
    <string>斜体</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="underline_action">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/underline.png</normaloff>:/icons/underline.png</iconset>
   </property>
   <property name="text">
    <string>下划线</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::TextHeuristicRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
